@kiva/kv-components/vue/stories/KvClassicLoanCard.stories.js

```diff
@@ -397,3 +400,32 @@ export const ClickableTags = story({
 	photoPath,
 	enableClickableTags: true,
 });
+
+export const SupportButton = story({
+	loanId: loan.id,
+	loan,
+	primaryButtonText: 'Support',
+	kvTrackFunction,
+	photoPath,
+});
+
+export const SupportAgain = story({
+	loanId: loan.id,
+	loan: {
+		...loan,
+		userProperties: { lentTo: true },
+	},
+	primaryButtonText: 'Support',
+	kvTrackFunction,
+	photoPath,
+});
+
+export const RemoveButton = story({
+	loanId: loan.id,
+	loan,
+	kvTrackFunction,
+	photoPath,
+	basketItems: [{ id: loan.id, __typename: 'LoanReservation' }],
+	showRemoveButton: true,
```

Comment:
I don't think this prop is implemented... I htink you need to pass in the `secondaryButtonHandler` to activate this feature

Reply:
done